addons/website_payment/static/src/snippets/s_donation/000.js

```diff
@@ -0,0 +1,123 @@
+/** @odoo-module **/
+
+import {_t} from 'web.core';
+import publicWidget from 'web.public.widget';
+
+publicWidget.registry.DonationSnippet = publicWidget.Widget.extend({
+    selector: '.s_donation',
+    events: {
+        'click .s_donation_btn': '_onClickPrefilledButton',
+        'input #s_donation_range_slider': '_onInputRangeSlider',
+    },
+
+    /**
+     * @override
+     */
+    async start() {
+        if (this.el.dataset.displayOptions) {
+            this.minAmount = this.el.dataset.minimumAmount;
+            const defaultAmount = this.el.dataset.defaultAmount;
+            this.$rangeSlider = this.$('#s_donation_range_slider');
+            this.$amountInput = this.$('#s_donation_amount_input');
+            if (this.$rangeSlider.length) {
+                this.$rangeSlider[0].min = this.minAmount;
+                this.$rangeSlider[0].max = this.el.dataset.maximumAmount;
+                this.$rangeSlider[0].step = this.el.dataset.sliderStep;
+                this.$rangeSlider.val(defaultAmount);
+                this._setBubble(this.$rangeSlider);
+            } else if (this.$amountInput.length) {
+                this.$amountInput[0].min = this.minAmount;
+                this.$('.s_donation_minimum_value').text(this.minAmount);
+            }
+        }
+        await this._displayCurrencies();
+        await _super(...arguments);
```

Comment:
;) 
https://github.com/odoo/odoo/pull/63133/files#diff-dbfc008cda51c8f3baf9dcf06dd50291181bde53cf0dfe173676da6c0c89d4bfR18

Reply:
Indeed, not sure how I missed it :shrug: I remembered looking for it too :man_facepalming: Why not starting with `await this._super(...arguments)` though? One line instead of two.

Reply:
@bso-odoo I think that's a part you did yourself. Is it right ?

Reply:
I just checked and it seems so. (between f13936 and 161a4e)
I guess this was done when handling [this comment](https://github.com/odoo/odoo/pull/63133#discussion_r682630967).
I guess I did lookup how async methods are awaiting for their super calls, and unfortunately stumbled on one of those 40+.